src/state/stores.js

```diff
@@ -1,21 +1,36 @@
 import { mapValues, pickBy } from "lodash";
 import { stringify } from "query-string";
-import { get, writable } from "svelte/store";
+import { writable, get } from "svelte/store";
 
 export const pageTitle = writable("");
 export const pageState = writable({});
 
-// synchronizes url state with page state
-export const updateURLState = (push = false) => {
-  const simplifiedState = mapValues(
-    pickBy(get(pageState), (v) => (typeof v !== "string" && v) || v.length > 0),
+// simplifies the page state (removing redundant values)
+// before updating the store
+const setPageState = (newState) => {
+  pageState.set(mapValues(
+    pickBy(
+      { ...get(pageState), ...newState },
+      (v) => (typeof v !== "string" && v) || v.length > 0
+    ),
     (v) => (typeof v === "boolean" ? +v : v)
   );
+};
```

Comment:
Looks like we're missing a `)` here.

Reply:
Also we shouldn't be doing the state simplification there! 🤦